vpr/src/device/rr_graph_view.h

```diff
@@ -258,7 +258,13 @@ class RRGraphView {
      * This API is very powerful and developers should not use it unless it is necessary, 
      * e.g the node type is unknown. If the node type is known, the more specific routines, `node_pin_num()`, 
      * `node_track_num()`and `node_class_num()`, for different types of nodes should be used.*/
+    /** @brief Return detailed routing segment information with a given id* @note The routing segments here may not be exactly same as those defined in architecture file. They have been
+     * adapted to fit the context of routing resource graphs.
+     */
 
+    inline const t_segment_inf& rr_segments(RRSegmentId seg_id) const {
+        return rr_segments_[seg_id];
+    }
```

Comment:
You inserted the code between the comments of ``node_ptc_num`` and declaration of ``node_ptc_num``
Please keep the comments and declaration close as other APIs do.

This will cause out-of-order in documentation: 

See details in https://vtr--1910.org.readthedocs.build/en/1910/

![image](https://user-images.githubusercontent.com/11499826/144102534-249ee4c0-5227-44d1-9e04-e265b0422854.png)